cmd/gaia/cli_test/test_helpers.go

```diff
@@ -305,6 +305,13 @@ func (f *Fixtures) QueryTxs(tags ...string) []tx.Info {
 	return txs
 }
 
+// QueryTxsWrong query txs with wrong parameters and compare expected error
+func (f *Fixtures) QueryTxsWrong(expectedErr error, page, limit int, tags ...string) {
```

Comment:
I think `QueryTxsInvalid` or `QueryTxsError` would provide a cleaner API, thoughts?

Reply:
I'm a fan of QueryTxsInvalid.